Lunch/ViewController.swift

```diff
@@ -7,21 +7,43 @@
 //
 
 import UIKit
+import Parse
 
 class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UITextFieldDelegate {
 
     @IBOutlet weak var lunchTableView: UITableView!
     @IBOutlet weak var lunchTextField: UITextField!
 
-    var lunchChoice: Int!
-    var items: [String] = []
+    lazy var refreshControl: UIRefreshControl = {
+        let refreshControl = UIRefreshControl()
+        refreshControl.addTarget(self, action: "handleRefresh:", forControlEvents: .ValueChanged)
+        return refreshControl
+    }()
+
+    var eateries: [String] = []
```

Comment:
You can also do `var eateries = [String]()` but I'm not sure which is better lol